### PR TITLE
Add Maint Tech ColMarTech

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Basically title, engi gaming

## Media

зображення

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      takeAll: CMStandard
+      entries:
+      - id: CMHandsInsulated
+      - id: CMHeadsetEngineer
+      - id: CMMRE
+        name: MRE
+      # map
+    - name: Helmet
+      choices: { CMHelmet: 1 }
+      entries: []
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml:30]:
```suggestion
      entries: []
      # Beret, white/orange/blue hardhat, welding helmet
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    - name: Suit
+      choices: { CMSuit: 1 }
+      entries: []
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml:33]:
```suggestion
      entries: []
      # Black/blue/Orange/Yellow hazard vest
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      - id: CMBackpackMarineTech
+      - id: CMSatchelMarineTech
+      - id: CMBackpackWelder
+      - id: RMCSatchelWelder
+      # Welder Chestrig
+    - name: Belt
+      choices: { CMBelt: 1 }
+      entries:
+      #- id:G8-A General Utility Pouch
+      - id: CMBeltUtilityFilled
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml:46]:
```suggestion
      - id: CMBeltUtilityFilled
      	name: M276 Toolbelt Rig (Full)
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      entries:
+      - id: CMPouchGeneralMedium
+      - id: CMPouchFirstAidInjectors
+        name: first-aid pouch (refillable injectors)
+      - id: CMPouchFirstAidSplintsGauzeOintment
+        name: first-aid pouch (gauze, ointment) # TODO RMC14 splints comma
+      - id: CMPouchFirstAidPills
+        name: first-aid pouch (pill packets)
+      - id: CMPouchConstruction
+      #- id: CMElectronicsPouchFull
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml:58]:
```suggestion
      #- id: CMElectronicsPouchFull
      	#name: Electronics Pouch (Full)
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      - id: CMPouchToolsFill
+        name: Tools pouch (Filled)
+      - id: CMPouchFlareFilled
+        name: Flare pouch (Filled)
+    - name: Mask
+      choices: { CMMask: 1 }
+      entries:
+      - id: CMMaskGas
+      - id: CMMaskCoif
+      #- id: CMRebreather #TODO: Make this
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml:68]:
```suggestion
      #- id: CMRebreather #TODO RMC14: Make this
```

**Review by QuietlyWhisper — DISMISSED** [review]

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      - id: CMPouchGeneralMedium
+      - id: CMPouchFirstAidInjectors
+        name: first-aid pouch (refillable injectors)
+      - id: CMPouchFirstAidSplintsGauzeOintment
+        name: first-aid pouch (gauze, ointment) # TODO RMC14 splints comma
+      - id: CMPouchFirstAidPills
+        name: first-aid pouch (pill packets)
+      - id: CMPouchConstruction
+      #- id: CMElectronicsPouchFull
+      	#name: Electronics Pouch (Full)
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/engineer.yml:62]:
```suggestion
      #name: Electronics Pouch (Full)
```

**Review by DrSmugleaf — APPROVED** [review]
janitor major victory

**Pull request merged**